examples/text-generation/README.md

```diff
@@ -356,6 +356,18 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
 --trim_logits \
 ```
 
+### Running FP8 models on single device
+
+Some bf16 models doesn't fit on one card due to hpu memory limitation, but in fp8 precision they do fit.
+As measurement is being calculated in bf16 precision, to be able to run fp8 model on single card you should use `unify_measurements` script.
+More information on usage of the unifier script can be found in fp8 Habana docs mentioned above.
+
+### CPU memory reduction on single card
+
+Some models can fit on HPU RAM but can't fit on the CPU RAM.
+When we run a model on single card and don't use deepspeed the '--disk_offload' flag allows to offload weights to disk during model quantization in HQT. When this flag is mentioned, during the quantization process, each weight first is offloaded from disk to CPU RAM, when quantized and finally brought to HPU RAM. This way not all the model is on the CPU RAM but only one weight each time.
+The usage of the flag is just to add it to the script.
```

Comment:
Unclear - which script? how to use - maybe give an example like in the flash attention. - parameter of what deepspeed/run-generation.py/something else...